docs/entropic-workflow.md

```diff
@@ -0,0 +1,36 @@
+# A workflow for Entropic
+
+This project has to this point been a close collaboration between two people who worked together for several years, and have an established workflow. If we want to succeed with this project, we'll need a successful workflow for collaborating asynchronously with people who don't know each other as well. I have some suggestions based on both things I've done with teams in the past and descriptions of other successful processes.
+
+In this one-pager I propose a workflow and discuss some of the tradeoffs.
+
+## Overview
+
+There is no process that will replace talking to people. Write a problem statement and proposed solution. Share it privately with one or more contributors and ask for feedback ahead of starting a public discussion. Be most formal when the problem is especially tricky or contentious.
+
+## A workflow
+
+Follow these steps:
+
+- Start a discussion about a problem you think needs to be solved. This might be a missing feature, or a an API that needs to be designed, or some other thing you think the project would benefit from.
+- This initial discussion should focus on *developing a clear problem statement* and *gathering information about the problem.* We can also discuss possible solutions & their tradeoffs.
+- Larger or more contentious discussions should have a moderator gently keeping people on track. If you're the proposer, you shouldn't moderate the discussion. This will free you to have opinions.
```

Comment:
Where can someone get a moderator from?

Reply:
Over time I would assume we would have members of the community who have committed to the code base to be available to act as moderators?

Reply:
Are we going to use the built-in issues for these type of discussions or would it be on whatever platform is going to be used to reach out to people like Discourse, Slack, Discord, ... ?

Reply:
@ceejbot if only github sponsorship was out of beta, we get set up entropic donations. I'd be happy to contribute in some way should you come up with a donation plan.

Reply:
@ceejbot Discourse offers free hosting for qualifying projects: https://free.discourse.group/